scripts/generate_dashboards.py

```diff
@@ -126,11 +128,15 @@ def get_dashboard_folder_name(self):
     def init_sae_session(self):
         (
             self.model,
-            self.sparse_autoencoder,
+            sae_group,
             self.activation_store,
         ) = LMSparseAutoencoderSessionloader.load_session_from_pretrained(self.sae_path)
+        # TODO: handle multiple autoencoders
+        self.sparse_autoencoder = sae_group.autoencoders[0]
```

Comment:
I had to pull out just the first autoencoder from the group here as well. Is this file also going to be removed in the future?